definitions/marts/bigquery_marts/npq/npq_enrolments.sqlx

```diff
@@ -23,9 +23,9 @@ config {
         la_name: "Name of local authority.",
         school_postcode: "Postcode of participant's school",
         course_name: "Text name of NPQ course.",
-        declaration_course_identifier: "Shorthand of NPQ course. This is sourced from npq_courses_latest table (not declarations) but it is used to match against declarations along with the application_ecf_id",
+        declaration_course_identifier: "Shorthand of NPQ course. This is sourced from npq_courses_latest table (not declarations) but it is used to match against declarations along with the application_ecf_id. It's not PII",

```

Comment:
Why is it important to note it's not PII?

Reply:
Whoops, put that on the wrong line. Meant to put it on ecf_user_id so that you can use that instead of TRN (which isn't really PII either but is a little more of an issue)